spec/account/account-manager.spec.js

```diff
@@ -51,32 +41,24 @@ describe("AccountManager", () => {
         otDateMock = require("../mock/ot-date-mock");
         promiseMock = require("../mock/promise-mock");
         randomMock = require("../mock/random-mock");
-        accountManager = require("../../lib/account/account-manager")(accountServiceFake, config, hotpFake, otDateMock, promiseMock, randomMock);
+        create = (config) => {
+            return require("../../lib/account/account-manager")(accountServiceFake, config, hotpFake, otDateMock, promiseMock, randomMock);
```

Comment:
Consider abbreviating the code by exposing a default configuration that `create` uses.

```
defaultConfig = { ... };
create = () => {
    return require("...")(accountServiceFake, defaultConfig, ...)
};
```